.gitignore

```
@@ -1,5 +1,6 @@
 node_modules
 .cache
 .parcel-cache
-dist/index.b4b6dfad.js
-dist/index.b4b6dfad.js.map
\ No newline at end of file
+index.b4b6dfad.js
+index.b4b6dfad.js.map
+index.b4b6dfad.js.css
```

This will ignore all the auto generated files in the `dist/` folder.
```suggestion
dist/*
```